src/__generated__/ArtworkSidebarBidAction_artwork.graphql.ts

```diff
@@ -12,6 +12,7 @@ export type ArtworkSidebarBidAction_artwork = {
         }) | null;
     }> | null;
     readonly id: string;
+    readonly _id: string;
```

Comment:
Did you manually edit this?

Reply:
yes i manually edited this to see if i could get feedback on the correct process... will run the compiler and fix this up, thanks for the tip!